inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/resources/.env.example

```diff
@@ -0,0 +1 @@
+SERVER_PORT=1000
```

Comment:
What server is meant here?

Reply:
I can't tell you exactly as I just copied the PDFAnno code over to inception.
There existed a server component in PDFAnno in version 0.4.1, however it was removed and not available in 0.5.0.
Maybe they forgot to delete this file.
Should I remove it?

Reply:
I guess if we don't need it, we should not have it in the codebase. I hope that we can remove more code that we don't need later - once we figured out that we don't need it ;)

Reply:
I can do that.
I was not sure whether the whole project should be copied over or only what is really required.
Should I remove unnecessary stuff in this issue / PR or should I open a new one?
I am still a bit uncertain about how small or large an issue / PR should be. :)

Reply:
This is less about how large/small a PR should be but rather about not having useless files lying around. So we should not commit/merge code/resources that we don't use.

Reply:
What is the best approach now to get rid of the files we don't want?
Should I just make another commit to this branch where the files are removed or is there a nicer and cleaner way?

Reply:
We can squash merge this pull request at the end I think.

Reply:
@wpmp what @Rentier said ;) Normally we do not squash, but in this case, I'd certainly squash - yes.